website/tosti/templates/tosti/base.html

```diff
@@ -53,29 +49,44 @@
                             Order now
                         </a>
                     </li>
+                </ul>
+            </div>
+            <div class="mx-auto order-0 navbar-collapse collapse position-relative">
+                <a class="navbar-brand" href="/"><img src="{% static 'tosti/svg/TOSTI-logo.svg' %}" height="60"/></a>
+            </div>
+            <div class="order-0 d-block d-md-none">
+                <a class="navbar-brand" href="/"><img src="{% static 'tosti/svg/TOSTI-logo.svg' %}" height="60"/></a>
```

Comment:
I still think it would be nice to make the logo larger and make it drop out of the menu bar a bit (but I understand that might be hard though)

Reply:
Rt